### ISSUE TITLE
A11y: Add checkmarks on filter badges when they are selected

### ISSUE THREAD

**Issue by marvieqa** [CodeForPhilly/vacant-lots-proj#300]
## Description

We should have another indicator that the filter badge is turned on or off. Currently, only color is used to denote whether it is selected or not. Color shouldn't be the only means to convey the meaning of the badges. 

WCAG Criterion: [1.4.1 Use of Color Level A](https://www.w3.org/WAI/WCAG22/quickref/#use-of-color)

I believe the Figma prototype used to have check icons, this but I don't see this anymore.

<details>
<summary> Screenshot </summary>

<img width="1472" alt="Screenshot 2024-01-30 at 17 59 38" src="https://github.com/CodeForPhilly/vacant-lots-proj/assets/102739972/4bb4ba8a-eeb7-4dae-8832-e2336b57a120">

</details>

## Acceptance Criteria

- badges should have a symbol or something as additional indication that it is selected.

**Comment by bacitracin** [comment]
PR: https://github.com/CodeForPhilly/vacant-lots-proj/pull/352
